### PR TITLE
Update Activity to include day

### LINKED ISSUE #38: Activity Sensor
I am encountering an issue where the activity sensor doesn't give the day attribute back in the sensor. It looks like either the api changed, and docs didn't or there is an issue with parsing them which causes issue.

### PR DESCRIPTION
**Pull request by maco2035**
This is adding the day to the Activity sensor attributes, It helps to prevent issues that I encountered from the day not showing up by default compared to other sensors. fixes #38

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -314,7 +314,7 @@ This sensor supports all the following monitored attributes:
 
 For a definition of all these variables, check [Oura's API](https://cloud.ouraring.com/v2/docs#operation/daily_activity_route_daily_activity_get).
 
-By default, the following attributes are being monitored: `active_calories`, `high_activity_time`, `low_activity_time`, `medium_activity_time`, `non_wear_time`, `resting_time`, `sedentary_time`, `score`, `target_calories`, `total_calories`.
+By default, the following attributes are being monitored: `active_calories`, `high_activity_time`, `low_activity_time`, `medium_activity_time`, `non_wear_time`, `resting_time`, `sedentary_time`, `score`, `target_calories`, `total_calories`, `day`.
```
Comment (nitobuendia) [inline README.md:317]: nit: The attributes are alphabetically ordered. Do you mind putting `day` between `active_calories` and `high_activity_time`?

File: custom_components/oura/sensor_activity.py [hunk 2]
```diff
@@ -22,6 +22,7 @@
     'score',
     'target_calories',
     'total_calories',
+    'day',
```
Comment (nitobuendia) [inline custom_components/oura/sensor_activity.py]: nit: The attributes are alphabetically ordered. Do you mind putting 'day' between 'active_calories' and 'high_activity_time'?

**Review by nitobuendia — CHANGES_REQUESTED** [review]
The code changes look good. Thanks for making these. I have added a small comment on each file, then we can merge :)

**Comment by maco2035** [comment]
@nitobuendia Thanks for reviewing my PR, and making this cool integration!

File: README.md (hunk 1 above)
Comment (nitobuendia) [reply README.md:317]: This was changed only for sensor_activity.py and not for `README`.

**Review by nitobuendia — APPROVED** [review]
One of the changes was not done, but approving as it's on the docs.

**Pull request merged**